genie-core/src/main/java/com/netflix/genie/core/jobs/workflow/impl/ClusterTask.java

```diff
@@ -56,10 +55,10 @@
      * Constructor.
      *
      * @param registry The metrics registry to use
-     * @param fts File transfer service
+     * @param fts      File transfer service
      */
     public ClusterTask(@NotNull final Registry registry,
```

Comment:
This method is not following our standard indentation style.
Since you're re-indenting it would have been nice to fix. (Not worth amending this commit just for this, leave as is is ok, just keep in mind for the future).

Reply:
I noticed that but this change in indentation happened by the formatter(command option L), so I left it as such